cvat-data/src/js/decode_video.js

```diff
@@ -91,7 +91,16 @@ self.onmessage = function (e) {
                 + 'bad video file, unpached jsmpeg';
             throw Error(message);
         }
-        postMessage({fileName : null, index : i, data: YCbCrToRGBA(...result)});
+        const rawImageData = {
+              data: YCbCrToRGBA(...result),
+              width: result[3],
+              height: result[4]
+            };
+        
+        const jpegImageData = jpeg.encode(rawImageData, 50);
```

Comment:
I don't think that 50 value is correct here. We already compressed the original quality on the server. Probably here you want to have mostly 100% quality.